.travis.yml

```diff
@@ -17,10 +17,7 @@ cache:
 install:
   # Ensure we have the latest Chrome stable.
   - google-chrome-stable --version
-  # if our e2e tests fail in the future it might be that we are not compatible
-  # with the latest puppeteer api so we probably need to run on chromimum
-  # @see https://github.com/GoogleChrome/lighthouse/pull/4640/files#r171425004
-  - export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1
+  - unset PUPPETEER_SKIP_CHROMIUM_DOWNLOAD
```

Comment:
I'm assuming can we remove this at some point?

Reply:
yeah, but can't be sure at what point. unless someone wants to merge master into ever outstanding PR branch after this lands.

It sure is unexpected that environment variables seem to persist.

Reply:
we might need to just clear the caches. If we keep this we should also leave a comment similar to the removed one

Reply:
Clearing cache didn't help, I tried that first.

Reply:
> environment variables seem to persist

I just don't see how this makes sense. It seems like people would be constantly running into it.

Reply:
Maybe b/c we should have use `env:` instead of a manual `export`?